tests/ops/integration_tests/test_execution.py

```diff
@@ -446,11 +446,8 @@ async def test_skip_collection_new_request(
         assert mongo_logs.filter_by(status="skipped").count() == 9
 
     @mock.patch("fides.api.task.graph_task.GraphTask.log_start")
+    @pytest.mark.usefixtures("use_dsr_2_0")
     @pytest.mark.asyncio
-    @pytest.mark.parametrize(
-        "dsr_version",
-        ["use_dsr_3_0", "use_dsr_2_0"],
-    )
     async def test_run_disabled_collections_in_progress(
```

Comment:
I am removing running this flaky test on DSR 3.0. This test was written for DSR 2.0 but it fails occasionally on CI. It is hard to reproduce when you run unit tests locally, you have to be running Python 3.8.x, and it will maybe fail 1/20 of the time.   I just don't think this is a good test here.